test/testother.cpp

```diff
@@ -3488,7 +3491,7 @@ class TestOther : public TestFixture {
         check("void foo(int a) {\n"
               "    if (a == a) { }\n"
               "}");
-        ASSERT_EQUALS("[test.cpp:2]: (style) Same expression on both sides of '=='.\n", errout.str());
+        ASSERT_EQUALS("[test.cpp:2] -> [test.cpp:2]: (style) Same expression on both sides of '=='.\n", errout.str());
```

Comment:
It seems to me to be more user friendly to just have 1 location when no variable is followed. Do you think that would be possible?

Reply:
I updated it to report the operator and not the expressions.